### DISCUSSION TITLE
Where is the Mitsumi proprietary CD-ROM option

### DISCUSSION THREAD

**Discussion by lazerode** [86Box/86Box#3686] [Q&A] [answered]
I noticed in builds that are close to the release of version 4.0 have the option of using Mitsumi proprietary CD-ROM drives missing. Is the ability to use Mitsumi proprietary CD-ROM drives removed? I am using the latest nightly ROM set.

**Comment by pankoza2-pl** [comment]
it wasn't even implemented in the first place, it was just a non-working stub so it got commented-out until 86Box actually adds Mitsumi CD-ROM emulation

**Comment by jriwanek** [comment, marked as answer]
It has now been implemented and will be available as part of 86Box 7.0
